lib/cldr-plurals/javascript_emitter.rb

```diff
@@ -84,6 +82,11 @@ def emit_value_set(value_set, operand, operator)
         end
       end
 
+      def emit_range_check(range, operand)
+        # a..b represents all *integers* between a and b, inclusive.
+        (range.start..range.finish).to_a.map { |v| "(#{emit(operand)} == #{v})" }.join(" || ")
```

Comment:
Ah that makes sense. What about something like this? It's not that easy to look at, but at least there wouldn't be tons of individual comparisons.

```javascript
(function(){var arr=[];for(i=11;i<=99;i++){arr.push(i)};return arr;})().indexOf(n) > -1
```